javascript/first-js-oop-project.md

```diff
@@ -0,0 +1,142 @@
+# Mini Project: Object Oriented Programming in JS
+
+### Projected Time
+
+Total: 65 min- 80min
+- Independent Practice: 15 min
+- Materials: 25 min
+- Lesson: 30 min
+
+### Prerequisites
+
+- [Java script lesson 7](../javascript/javascript-7-oop.md)
+
+
+### Objectives
+
+- To use the new JS object-oriented programming skills.
+- To create a few event objects and push them to an array.
+- Use jQuery to display the list of events to an html page.
+
+### Specific Things To Teach
+
+- Creating event array.
+- Iterating through the array.
+- Displaying array items in HTML page.
+
+### Materials
+
+- [Documentation on each statement of jQuery](http://api.jquery.com/jquery.each/)
+- [Arrays in javascript](https://www.w3schools.com/js/js_arrays.asp)
+- [.html function of jQuery](https://www.w3schools.com/jquery/html_html.asp)
+
+### Lesson
+
+- The below example is the just a part of a mini project that will help you to learn basic concept about arrays in javascript and iterating through the arrays. The below code will allow you to learn about displaying the array objects in the web pages.
+- Once you are done with the code, run the index.html file in your browser to check the result.
+- Keep on refreshing the browser to keep track of the changes you make everytime.
+
```

Comment:
The steps are confusing. Why is there an unordered list, then an ordered list and again an unordered list in the same level? If they are of different levels, please align them accordingly.

Reply:
the info under each step should be indented.  Right now, with it all on the same level, we have no idea what is part of which step.

Reply:
@alodahl  I have modified the indentation. I hope it works now.